component/rbac.jsonnet

```diff
@@ -144,6 +144,64 @@ local cr_admission_controller = kube.ClusterRole('system:vpa-admission-controlle
   ],
 };
 
+//
+// Cluster Roles which get aggregated to the standard roles
+//
+local aggregated_view = kube.ClusterRole('syn:vertical-pod-autoscaler:view') {
+  metadata+: {
+    labels+: {
+      'rbac.authorization.k8s.io/aggregate-to-admin': 'true',
+      'rbac.authorization.k8s.io/aggregate-to-edit': 'true',
+      'rbac.authorization.k8s.io/aggregate-to-view': 'true',
+      'rbac.authorization.k8s.io/aggregate-to-cluster-reader': 'true',
+    },
+  },
+  rules: [
+    {
+      apiGroups: [ 'autoscaling.k8s.io' ],
+      resources: [ 'verticalpodautoscalers' ],
+      verbs: [ 'get', 'list', 'watch' ],
+    },
+  ],
+};
+
+local aggregated_edit = kube.ClusterRole('syn:vertical-pod-autoscaler:edit') {
+  metadata+: {
+    labels+: {
+      'rbac.authorization.k8s.io/aggregate-to-admin': 'true',
+      'rbac.authorization.k8s.io/aggregate-to-edit': 'true',
+    },
+  },
+  rules: [
+    {
+      apiGroups: [ 'autoscaling.k8s.io' ],
+      resources: [ 'verticalpodautoscalers' ],
+      verbs: [
+        'create',
+        'delete',
+        'deletecollection',
+        'patch',
+        'update',
+      ],
+    },
+  ],
+};
+
+local aggregated_cluster_reader = kube.ClusterRole('syn:vertical-pod-autoscaler:cluster-reader') {
+  metadata+: {
+    labels+: {
+      'rbac.authorization.k8s.io/aggregate-to-cluster-reader': 'true',
+    },
+  },
+  rules: [
+    {
+      apiGroups: [ 'autoscaling.k8s.io' ],
+      resources: [ 'verticalpodautoscalercheckpoints' ],
+      verbs: [ 'get', 'list', 'watch' ],
+    },
+  ],
+};
+
```

Comment:
What's the reasoning behind aggregating this to the cluster reader, the CRD also seems to be namespaced?  Can a user crash or otherwise mess with the autoscaler by modifying its state?

Reply:
From what I could find, this resource is not intended to be used/modified by users, it's a mechanism for VPA to cache its internal state.

Reply:
Yeah, I think you're right. I don't have strong feelings if we should expose this to users or not, so we can leave it as is